### ISSUE TITLE
Any plan to change dust to 136sat

### ISSUE THREAD

**Issue by gitzhou** [AustEcon/bitsv#74]
Hi Aust,

The dust policy has been changed after Node 1.0.5, now the minimum output is 136sat, according to the [release note](https://bitcoinsv.io/2020/09/16/bitcoin-sv-node-software-important-upgrade-to-v1-0-5/).

![image](https://user-images.githubusercontent.com/1585505/96691606-ab5eed00-13b7-11eb-96d7-63d538168e5e.png)

I broadcast 136sat and 135sat on WoC.

- 136sat, OK

![image](https://user-images.githubusercontent.com/1585505/96693022-5623db00-13b9-11eb-9631-79a4e6618136.png)

https://whatsonchain.com/tx/827ad66201f7eb74e6ef6fc2214517a17946712d06cbf2090f4ec577d47cb3d5

- 135sat, failed

```
0100000001cb163b2fbf0b90b0b6da01515e39cf5c9ea87b207028185d9c4ce49d69da7446000000006a473044022045b0c94a4aee3526c1b5fb64cecae014762b8b19c5bd202a83082b845737f5e0022077f079a5eeb8ff86e80c9580d036f51960ed04b35c392640d11d3dc5db1d74214121028f4d50294ad3a60703fd8f48be59f2543988eff880a7cb28d9940ee68a992cdbfeffffff0286000000000000001976a9146127e47d5f28d322ef454548af46d9b5b652d68888ac27020000000000001976a914b49bd472ff3e2cdd3b1dd472f114de1bd5ab9fa488ac93090a00
```

![image](https://user-images.githubusercontent.com/1585505/96693064-5e7c1600-13b9-11eb-862d-198b3c027b6d.png)

Any schedule to follow this? Thank you!
https://github.com/AustEcon/bitsv/blob/72a48a36fbf2e9abd0c32ba2316f83cf169131c8/bitsv/transaction.py#L18

**Comment by AustEcon** [comment]
Thanks for this. As long as it always relays every time and confirms then no objections from me.

I guess I will put it on the todo list then 😉

**Comment by gitzhou** [comment]
Yes, agree. It still needs some time to wait for the majority of nodes to upgrade.
